.travis.yml

```diff
@@ -0,0 +1,62 @@
+# Travis configuration file for downloading and building OP-TEE according to the
+# instructions in build.git
+language: bash
+
+notifications:
+  email:
+    recipients:
+      - op-tee@linaro.org
+    on_success: change
+    on_failure: always
+
+dist: trusty
+sudo: required
+group: beta
+
+cache:
+  #ccache: true
```

Comment:
I'll make the change when adding the tag. I'm gonna take the liberty to make yet one more change and that is to add my self to the email section. I recall reading that to receive emails one has to have the account register. So .. until we've figured out how to do that I'll have my own email there.